debian

```diff
@@ -12,8 +12,8 @@
 
 NAME="example"
 PATH="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
-APPDIR="/usr/bin"
-APPBIN="python"
+APPDIR="/"
```

Comment:
Does this make sense? Now since you call the executable via absolute path, why do you need to change into `/` directory?